src/jit-format/jit-format.cs

```diff
@@ -580,52 +589,61 @@ public static bool RunClangTidy(List<string> filenames, string compileCommands,
                         Console.WriteLine("\tclang-tidy " + tidyFix + " -checks=-*," + checks + fixErrors + " -header-filter=.* -p " + compileCommands + " " + filename);
                     }
 
-                    Process process = new Process();
-                
-                    // Configure the process using the StartInfo properties.
-                    process.StartInfo.FileName = "clang-tidy";
-                    process.StartInfo.Arguments = tidyFix + "-checks=-*," + checks + fixErrors + " -header-filter=.* -p " +
-                        compileCommands + " " + filename;
-                    process.StartInfo.UseShellExecute = false;
-                    process.StartInfo.RedirectStandardOutput = true;
-                    process.StartInfo.RedirectStandardError = true;
-                    process.StartInfo.CreateNoWindow = true;
-                    process.Start();
-
-                    string stdoutx = process.StandardOutput.ReadToEnd();         
-                    string stderrx = process.StandardError.ReadToEnd();             
-                    process.WaitForExit();
-
-
-                    if ((stdoutx.Contains("warning:") || (!ignoreErrors && stdoutx.Contains("error:"))))
+                    try
                     {
-                        if (verbose)
+                        Process process = new Process();
```

Comment:
This is new. You didn't miss it. Sorry.

Let me look into this simpler command invocation wrapper.